### ISSUE TITLE
Aqual Quintessence quest chain issue

### ISSUE THREAD

**Issue by godnesso** [chromiecraft/chromiecraft#2654]
### What client do you play on?

enGB

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59
- [X] 60

### Current Behaviour

Quests offered by Duke Hydraxis in Azshara can be accepted outside of their intended order. "The Molten Core" and "Agent of Hydraxis" can be accepted without needing to do the quests in order.

### Expected Blizzlike Behaviour

Quest order should be:
1. Poisoned Water & Stormers and Rumblers
2. The Eye of Emberseer
3. The Molten Core
3. Agent of Hydraxis
4. Hands of the Enemy
5. A Hero's Reward

### Source

https://classic.wowhead.com/guides/hydraxian-waterlords-reputation-wow-classic#hydraxian-waterlords-quests

### Steps to reproduce the problem

Go to Lord Hydraxis in Azshara.

### Extra Notes

https://wowgaming.altervista.org/aowow/?npc=13278

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/c70cc855ff3d2724a0e0a3866b6a43db1f2faddb

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Josarth** [comment]
true.. i completed the first one and there is no chain avalaible

**Comment by neifion-00000000** [comment]
![image](https://user-images.githubusercontent.com/91289495/147209376-b48d51c2-914d-41ae-b4d2-aa6b629c2649.png)
I've also experienced it.

Interestingly, accepting either The Molten Core or Agents of Hydraxis closes the other.

**Comment by Shineslippers** [comment]
Thank you for the report. I guess this issue is already fixed and fix should be already loaded to CC.

**Comment by neifion-00000000** [comment]
thought I did this yesterday but must not have whoops, uh this issue wasn't properly fixed, I am currently on the hands quest with no way to go backwards

**Comment by Shineslippers** [comment]
@neifion-00000000 i do not understood you

**Comment by neifion-00000000** [comment]
> @neifion-00000000 i do not understood you

sorry I was editing it, is that better?

**Comment by Shineslippers** [comment]
Well, that's how it should be. Hand is last quest, If you done it before fix you have to talk with GM he can remove quests.
![Capture](https://user-images.githubusercontent.com/85256266/148137019-3b6da1a8-9439-42dd-af40-9db69537bd7c.PNG)

**Comment by neifion-00000000** [comment]
I did it after the fix, essentially I skipped The Molten Core, that's not correct.

**Comment by Shineslippers** [comment]
@Nyeriah can you look to it.

**Comment by neifion-00000000** [comment]
I'm unsure how it is if there has been changes, but originally Agents of Hydraxis was open before Eye of Emberseer was completed.  

When I checked last and opened a ticket (about a day and a half ago, I was told to report here and thought I did too but realized just now I didn't--anyway--), it was open and Eye of Emberseer is the last quest in the chain I completed in the normal fashion.  The Molten Core was open as well, as it should be.  By picking up Agents of Hydraxis, The Molten Core closes, and by completing it, the chain continues and the Hands quest opens.
